src/target/stm32f1.c

```diff
@@ -257,7 +257,11 @@ bool at32fxx_probe(target_s *t)
 	return false;
 }
 
-/* mm32l0 flash write */
+/* mm32l0 flash write
```

Comment:
The codebase multi-line comment style is
```c
/*
 * first line
 * second line
 */
```

Please alter this to match and drop the blank line after the comment so IDEs attach the comment to the function properly.

Reply:
ok, done.

Reply:
It's still missing the *'s on the beginning of each comment line, but it's close enough that we'll merge and next time we touch stm32f1.c we'll run a fixup pass.